canonical-websites/maas.io
# Mitigate feed errors by using the new feeds module

- Use the new feed module that is also used in www.ubuntu.com
- Driveby: Use the new redirects module too

QA
--

```
./run
```

- Go to `127.0.0.1:8006` and check "The latest MAAS news" section has populated.
- Open `templates/index.html` and change the feed URL to `https://example.com`, see that the homepage doesn't break but shows an error where the feed should be.
- Go to `127.0.0.1:8006/get-started` to check it redirects to `/install` properly
